source/module_io/read_input_item_deepks.cpp

```diff
@@ -1,4 +1,6 @@
 #include "module_base/constants.h"
+#include "module_base/global_variable.h"
```

Comment:
input depends on global_variable?

Reply:
Previously considered adding the GlobalV parameter, but it has been commented out. The dependency can be removed directly here, i will remove it now.